docs/API.md

```diff
@@ -256,9 +246,9 @@ TrackPlayer.updateOptions({
     stopIcon: require('./stop-icon.png'),
     previousIcon: require('./previous-icon.png'),
     nextIcon: require('./next-icon.png'),
-    icon: require('./notification-icon.png'), // The notification icon
+    icon: require('./notification-icon.png')
 });
 ```
 
 ### Notes
-* Due to the notification not being swipable, we highly recommend you to have a stop button in the notification. The button should `destroy()` the player.
\ No newline at end of file
+* Because you can not stop the player service by swiping the Android notification, we highly recommend you to have a stop button in the notification. The button should `destroy()` the player.
```

Comment:
@guichaguri, is there a simple explanation we could include here for why the notification cannot be swipable?

Reply:
Basically the Android service requires a notification in order to be in the "foreground" state (allowing it to use more system resources without being killed), and that notification is forced to be "on going" (not swipable).

A stop button would be an intuitive and easy to implement solution to allow the user to get rid of the player. I've seen some apps destroying the player when it is closed while paused, which also works pretty well.

Reply:
Okay, I extended this note to include your explanation.